app/scss/checkbox.scss

```diff
@@ -0,0 +1,53 @@
+@import "variables.scss";
+
+.c-svg__check {
+  fill: #fff;
+}
+.c-list-item {
+  top: 5rem;
+  position: relative;
+  list-style: none;
+}
+.c-list-item__input {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  padding: 0 0 0 6rem;
+}
+
+.c-list-item__input input {
```

Comment:
Why do you need to call `input` on `.c-list-item__input`? Aren't they the same thing?